common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala

```diff
@@ -129,7 +130,7 @@ object CuratedRecipe {
       ),
       published = Some(
         ChangeRecord(
-          date = r.publicationDate.toInstant.toEpochMilli,
+          date = DateTime.now.toInstant.getMillis,
```

Comment:
@LATaylor-guardian @clloyd This is the [current documentation](https://github.com/guardian/content-atom/blob/master/thrift/src/main/thrift/contentatom.thrift#L38-L45) 
```thrift
  /** the latest change to the content atom */
  1: optional shared.ChangeRecord lastModified

  /** the atom creation event */
  2: optional shared.ChangeRecord created

  /** the atom publication event (if published) */
  3: optional shared.ChangeRecord published
```
Is `published` the first publication date or should it be updated each time the atom is `published`?